packages/components/file-card/src/file-card.js

```diff
@@ -49,9 +58,10 @@ customElementDefineHelper(
 
 		get cardType() {
 			const cardTypes = {
-				uploading: 'focus',
-				download: 'default',
-				failed: 'failed'
+				//TODO: Import card types from card component
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1135)